pkg/controller/syncsetinstance/syncsetinstance_controller_test.go

```diff
@@ -1503,3 +1503,40 @@ func computeHash(data interface{}) string {
 	}
 	return fmt.Sprintf("%x", md5.Sum(b))
 }
+
+func TestSSIReApplyDuration(t *testing.T) {
+	startProbe := time.Now()
+
+	tenMinutesAgo := time.Unix(metav1.NewTime(time.Now().Add(-10*time.Minute)).Unix(), 0)
```

Comment:
The `Unix` cruft is not necessary and will cause flakiness in the test.

Reply:
Removed.